File: Módulo 01/Lista Exercícios 05/classes/Retangulo.js
class Retangulo{
    largura
    altura
    constructor(largura, altura){
        this.largura = largura
        this.altura = altura
    }

    calcularArea(){
        return this.largura * this.altura
    }
}

const arrayRetangulos = []

arrayRetangulos.push(new Retangulo(5, 8))
arrayRetangulos.push(new Retangulo(3, 9))
arrayRetangulos.push(new Retangulo(4, 7))
arrayRetangulos.push(new Retangulo(9, 14))
arrayRetangulos.push(new Retangulo(6, 2))
arrayRetangulos.push(new Retangulo(7, 8))
arrayRetangulos.push(new Retangulo(8, 11))
arrayRetangulos.push(new Retangulo(9, 7))


for(i = 0; i < arrayRetangulos.length; i++){
    console.log("A área do retângulo " + (i+1) + " é " + arrayRetangulos[i].calcularArea())
}

File: Módulo 02/Semana 05/Exercicios-Modulo02-Semana05/EditorFotos/Retangulo.cs
using System;

namespace EditorFotos {
    public class Retangulo {
        int LadoA { get; set; }
        int LadoB { get; set; }

        public Retangulo(int ladoA, int ladoB) {
            LadoA = ladoA;
            LadoB = ladoB;
        }

        public void calculaArea(int ladoA, int ladoB) {
            Console.WriteLine($"A área do retângulo é {ladoA * ladoB}");
        }

        public void calculaPerimetro (int ladoA, int ladoB) {
            Console.WriteLine($"O perímetro do retângulo é {(ladoA * 2) + (ladoB * 2)}");
        }
    }
}
